definitions/checks/unread_mail.rb

```diff
@@ -0,0 +1,40 @@
+class Checks::UnreadMail < ForemanMaintain::Check
+  metadata do
+    label :unread_mail
+    description 'Check for unread mail for root user.'
+    tags :default
+  end
+
+  attr_reader :file_path
+
+  def initialize
+    @file_path = '/var/spool/mail/root'
```

Comment:
@gnurag , 
Done with the suggested changes. 
Instead of hard-coded path, value of `MAIL_DIR` is set as a mailbox path.
This value of `MAIL_DIR` is retrieved from config file `/etc/login.defs` .